ddmock/src/main/java/com/dd/DDMock.kt

```diff
@@ -69,6 +69,9 @@ object DDMock {
         when (extension) {
             "json" -> return MediaType.parse(MockEntry.CONTENT_TYPE_APPLICATION_JSON)
             "pdf" -> return MediaType.parse(MockEntry.CONTENT_TYPE_APPLICATION_PDF)
+            "jpeg", "jpg", "JPG", "JPEG" -> return MediaType.parse(MockEntry.CONTENT_TYPE_IMAGE_JPEG)
```

Comment:
Making the check case-insensitive should be somewhere shared in this function or somewhere in the flow into this function so that we don't need to manually check upper and lowercase. Also this doesn't cover mixed case e.g. `file.Jpeg`